src/content/cookbook/plugins/play-video.md

```diff
@@ -85,6 +85,25 @@ The `video_player` plugin can only play asset videos in iOS simulators.
 You must test network-hosted videos on physical iOS devices.
 :::
 
+### MacOS
+
+If you are using network-based videos, you will need to [add the
```

Comment:
**issue:** Write in active voice per [GSG](https://developers.google.com/style/voice). 

```suggestion
If you use network-based videos, [add the
```